latex/Dockerfile

```diff
@@ -29,8 +29,8 @@ EOF
 #* Use the `kinetic` variant of `buildpack-deps` as the base image, since it includes 
```

Comment:
**documentation:** the mention of `kinetic` here should either be updated or removed.

Reply:
thanks for catching this! 😃